paperqa/core.py

```diff
@@ -71,14 +75,25 @@ async def map_fxn_summary(
     citation = text.name + ": " + text.doc.formatted_citation
     success = False
 
-    if prompt_runner:
-        llm_result = await prompt_runner(
-            {"question": question, "citation": citation, "text": text.text}
-            | (extra_prompt_data or {}),
-            callbacks,
-            "evidence:" + text.name,
+    if summary_llm_model and prompt_templates:
+        data = {
+            "question": question,
+            "citation": citation,
+            "text": text.text,
+            **(extra_prompt_data or {}),
+        }
```

Comment:
```suggestion
        data = {
            "question": question,
            "citation": citation,
            "text": text.text
        } | (extra_prompt_data or {}),
```

Previously we could pass `text` in `extra_prompt_data` and override, but now we can't.

Let's avoid doing niche breaking changes like this within a refactor PR